<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WindowNes</class>
 <widget class="QWidget" name="WindowNes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Устройство не подключенно</string>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>950</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QRadioButton" name="radioButton_8">
          <property name="styleSheet">
           <string notr="true">QRadioButton::indicator{

	width: 13px;
	height: 13px;

}

QRadioButton::indicator::unchecked{

 image: url(://images/circle_gray.png);

}

QRadioButton::indicator::checked{

 image: url(://images/circle_zel.png);

}</string>
          </property>
          <property name="text">
           <string>20 МГц</string>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
          <property name="autoRepeat">
           <bool>false</bool>
          </property>
          <property name="autoExclusive">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_19">
          <property name="styleSheet">
           <string notr="true">QRadioButton::indicator{

	width: 13px;
	height: 13px;

}

QRadioButton::indicator::unchecked{

 image: url(://images/circle_gray.png);

}

QRadioButton::indicator::checked{

 image: url(://images/circle_zel.png);

}</string>
          </property>
          <property name="text">
           <string>10 МГц</string>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
          <property name="autoExclusive">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>1550</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QRadioButton" name="radioButton_21">
          <property name="styleSheet">
           <string notr="true">QRadioButton::indicator{

	width: 13px;
	height: 13px;

}

QRadioButton::indicator::unchecked{

 image: url(://images/circle_gray.png);

}

QRadioButton::indicator::checked{

 image: url(://images/circle_zel.png);

}</string>
          </property>
          <property name="text">
           <string>20 МГц</string>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <property name="autoExclusive">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_24">
          <property name="styleSheet">
           <string notr="true">QRadioButton::indicator{

	width: 13px;
	height: 13px;

}

QRadioButton::indicator::unchecked{

 image: url(://images/circle_gray.png);

}

QRadioButton::indicator::checked{

 image: url(://images/circle_zel.png);

}</string>
          </property>
          <property name="text">
           <string>10 МГц</string>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
          <property name="autoExclusive">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>2150</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QRadioButton" name="radioButton_25">
          <property name="styleSheet">
           <string notr="true">QRadioButton::indicator{

	width: 13px;
	height: 13px;

}

QRadioButton::indicator::unchecked{

 image: url(://images/circle_gray.png);

}

QRadioButton::indicator::checked{

 image: url(://images/circle_zel.png);

}</string>
          </property>
          <property name="text">
           <string>20 МГц</string>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
          <property name="autoRepeat">
           <bool>false</bool>
          </property>
          <property name="autoExclusive">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_29">
          <property name="styleSheet">
           <string notr="true">QRadioButton::indicator{

	width: 13px;
	height: 13px;

}

QRadioButton::indicator::unchecked{

 image: url(://images/circle_gray.png);

}

QRadioButton::indicator::checked{

 image: url(://images/circle_zel.png);

}</string>
          </property>
          <property name="text">
           <string>10 МГц</string>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
          <property name="autoExclusive">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
